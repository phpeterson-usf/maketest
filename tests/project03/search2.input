search 7 7 8 9 10
