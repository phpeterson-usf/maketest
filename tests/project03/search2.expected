C: 0
Assembly: 0
